src/Dialog/DialogBottomSheet.tsx

```diff
@@ -0,0 +1,318 @@
+import React, {useEffect, useRef, useState, PropsWithChildren, MouseEvent, TouchEvent, ChangeEvent} from 'react'
```

Comment:
We should add unit tests for this component. We would test things like whether the `onClose` prop is called when passed. You can use `npm run test:coverage -- src/Dialog/DialogBottomSheet.tsx` to get ideas for what we might want to test for.

Reply:
I can try and help too. Just push up your changes.

Reply:
Done